shell/platform/fuchsia/flutter/tests/integration/embedder/child-view/lib/child_view.dart

```diff
@@ -4,10 +4,10 @@
 
 import 'dart:ui';
 
-TestApp app;
-
 void main(List<String> args) {
-  app = TestApp();
+  print('child-view: starting');
```

Comment:
Just confirming we want to keep this

Reply:
It was quite helpful in debugging the test, since it allows one to know if the dart VM for child-view/parent-view has started execution  or not

Reply:
sg thanks